### PR TITLE
En 94 - Cascade on employee deletion

### PR DESCRIPTION
**Pull request by jauday**
# Description :pencil:

When an employee is deleted, his contracts and assignments must also be marked as deleted, and the last modified file would be updated.
The same process workflow when an employee is deactivated, both his contracts and assignments would be deactivated.

## Link to ticket :link:

https://entropyteam.atlassian.net/browse/EN-94

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? :microscope:

Manual testing through the UI

### CODE REVIEW

**Review by FacuOrellana — APPROVED** [review]
nice work :celebrate:

**Review by FedericoA15 — APPROVED** [review]
good job!

File: src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java [hunk 1]
```diff
@@ -105,14 +106,65 @@ public EmployeeDto create(EmployeeDto employeeDto) {
...
+            });
+            contractRepository.saveAll(employeeContracts);
+
+            employeeAssignments.forEach(assignment -> {
+                assignment.setActive(false);
+                assignment.setEndDate(LocalDate.now());
+            });
+            assignmentRepository.saveAll(employeeAssignments);
+        }
+
```
Comment (agustinb4) [inline src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java:128]: minor: remove this new lines to compact the code.

**Review by agustinb4 — APPROVED** [review]
Looks good!

**Pull request closed without merging**